### DISCUSSION TITLE
why the  enzyme relative amount is not correct in  version 11.1

### DISCUSSION THREAD

**Discussion by dongji1982** [Open-Systems-Pharmacology/Forum#1474] [Announcement]
![image](https://user-images.githubusercontent.com/81948560/224325956-c9cd94e4-b5ce-4a3b-a5d5-9a6a4fec2da8.png)

**Comment by PavelBal** [comment]
Could you please explain what you mean "why is it not corrected"?

**Comment by dongji1982** [comment]
Thanks， the enzyme acount is 0,  thanks  a lot. I have find a solution to set the enzyme amout  correct.. the New version setting is a little different.

从 Windows 版邮件发送

发件人: Pavel Balazki
发送时间: 2023年3月10日 21:41
收件人: Open-Systems-Pharmacology/Forum
抄送: dongji1982; Author
主题: Re: [Open-Systems-Pharmacology/Forum] why the enzyme relative amountis not correct in version 11.1 (Discussion #1474)

Could you please explain what you mean "why is it not corrected"?
